src/components/table/z-td/index.tsx

```diff
@@ -0,0 +1,97 @@
+import {Component, Element, Event, EventEmitter, Host, Prop, State, Watch, h} from "@stencil/core";
+import {ButtonVariant, ControlSize} from "../../../beans";
+
+/**
+ * ZTd component.
+ * @slot - ZTd content.
+ */
+@Component({
+  tag: "z-td",
+  styleUrl: "styles.css",
+  shadow: true,
+})
+export class ZTd {
+  @Element() host: HTMLZTdElement;
+
+  /**
+   * Number of columns that the cell should span.
+   */
+  @Prop()
+  colspan: number;
+
+  /**
+   * Whether the cell should stick.
+   */
+  @Prop({reflect: true})
+  sticky = false;
+
+  /**
+   * Enables the contextual menu.
+   * If true, a contextual menu button will be shown on hover.
+   */
```

Comment:
Attualmente ci sono casi in cui l'indicatore del contextual menu viene sempre mostrato, per esempio nella tab ATTIVITA' del registro. La logica di farlo comparire on hover è solo sulla tab VOTI, quindi credo vadano permesse entrambe le modalità

Reply:
ma come lo fate lato prodotto? guardando il codice vecchio non trovo nessuna implementazione di questa doppia casistica

Reply:
Attualmente vedo che viene creata una cella a se, dove viene inserito solo il contextual menu. Se la lasciamo così, forse va un po' in contrasto con il refactoring fatto. Magari va presa in considerazione come casistica e gestita nel componente?

Reply:
non dovrebbe andare in contrasto. nei z-td/z-th si può mettere quello che si vuole, quindi se il prodotto ha bisogno di mettere dei bottoni dentro, può farlo tranquillamente.
comunque possiamo sempre ridiscutere con tutti il comportamento del bottone e trovare la soluzione più comoda